securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -227,6 +227,13 @@ responseMode::
 
 flow::
     Flow passed in init.
+    
+adapter::
+    Allows to override way how redirects and other browser related functionalities will be handled by library.
+    Possible options:
+    * "default" - using browser api for redirects (used by default)
+    * "cordova" - will be used automatically when library is working in cordova ecosystem. When specifying this option library will try to use InAppBrowser cordova plugin to load keycloak login/registration pages.
+    * custom - function that allows to implement custom adapter (only for advanced use cases)
```

Comment:
adapter::
    Allows you to override the way that redirects and other browser-related functions will be handled by the library.
    Available options:
    * "default" - the library uses the browser api for redirects (this is the default)
    * "cordova" - the library will try to use the InAppBrowser cordova plugin to load keycloak login/registration pages (this is used automatically when the library is working in a cordova ecosystem)
    * custom - allows you to implement a custom adapter (only for advanced use cases)